docs/meshstack.email.md

```diff
@@ -6,7 +6,7 @@ title: Email
 For all email based communication (e.g. for [customer registration](./meshstack.onboarding.md#customer-registration) or [customer user invitations](./meshstack.onboarding.md#customer-user-invitations)) meshStack will use the configured SMTP server.
 
 <!--snippet:mesh.meshfed.mail-->
-
```

Comment:
f: don't put text into <!--snippet blocks, this will get replaced by tooling when updating dhall snippets against the meshStack config model